docs/pages/docs/form/picker.mdx

```diff
@@ -0,0 +1,210 @@
+import { Callout, Tabs } from 'nextra/components';
+import { Widget } from "../../../components/widget.tsx";
+import LinkBadge from "../../../components/link-badge/link-badge.tsx";
+import LinkBadgeGroup from "../../../components/link-badge/link-badge-group.tsx";
+
+# Picker
+
+A generic picker that allows an item to be selected. It is composed of one or more wheels, optionally, with separators
+between those wheels.
+
+The picker supports arrow key navigation:
+  * Up/Down arrows: Increment/decrement selected value
+  * Left/Right arrows: Move between wheels
+
+Recommended for touch devices.
```

Comment:
I played around with it on the browser, it feels super rough with the scroll wheel. Is there anyways we can adjust that?

Eg. one scroll notch on my mouse results in wheel skipping a few values.